App/Redux/AuthRedux.ts

```diff
@@ -20,6 +20,9 @@ const actions = {
   logInRequest: createAction('LOG_IN_REQUEST', resolve => {
     return (username: string, password: string) => resolve({ username, password })
   }),
+  logOutRequest: createAction('LOG_OUT_REQUEST', resolve => {
```

Comment:
Don't see any 'log out succes' action, but do we need it so we can clear out the `CafeTokens` stored in `AuthRexux`?

Reply:
I didn't complete it, figured it needs its own PR later. I commented out the Lock Screen button in the settings for now. The reason was that we need a few things to make it work,

- [ ] Goodbye screen (optional)
- [ ] When logged out after loggin in, we might want to skip those three intro screens before the login
- [ ] We currently force the user to retake their profile pic after any login, we prob want to skip that or at least ensure it works